libsolidity/lsp/LanguageServer.cpp

```diff
@@ -146,14 +146,30 @@ Json::Value LanguageServer::toJson(SourceLocation const& _location)
 void LanguageServer::changeConfiguration(Json::Value const& _settings)
 {
 	m_settingsObject = _settings;
+	Json::Value jsonIncludePaths = _settings["include-paths"];
+	if (jsonIncludePaths && jsonIncludePaths.isArray())
+	{
+		vector<boost::filesystem::path> includePaths;
+		int typeFailures = 0;
+		for (Json::Value const& jsonPath: jsonIncludePaths)
+		{
+			if (jsonPath.isString())
+				includePaths.emplace_back(boost::filesystem::path(jsonPath.asString()));
+			else
+				typeFailures++;
+		}
+		if (typeFailures)
```

Comment:
I think this would be much more readable. From the name it looks like a boolean but it isn't.
```suggestion
		if (typeFailureCount > 0)
```